### PR TITLE
Feat/readme update

### PR DESCRIPTION
**Pull request by Victor-Salomon**
Added the process to support a new Parachain

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -1,36 +1,68 @@
...
+- Bridge ERC20 tokens between Ethereum and Polkadot - Powered by Snowbridge
+- [Soon]: Seamless XCM Transfers between all Polkadot parachains
 
-To learn more about Next.js, take a look at the following resources:
+<br/>
+Turtle is developed with Typescript, Nextjs, and React.
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+## How to support a new parachain
```
Comment (NunoAlexandre) [inline app/README.md:36]: @Victor-Salomon I wouldn't place this here. This is developer-only content so I suggest that we create a `CONTRIBUTING.md` or a `DEVELOPING.md` file with this kind of info

**Review by NunoAlexandre — CHANGES_REQUESTED** [review]
> @Victor-Salomon I wouldn't place this here. This is developer-only content so I suggest that we create a `CONTRIBUTING.md` or a `DEVELOPING.md` file with this kind of info 

🙏

**Comment by noahjoeris** [comment]
I agree with Nuno here ☝️  Although it's also a good thing to replace the default nextjs readme 😄

File: app/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  - tokens array
+  - add new routes and select the according SDK that supports transfers for this route
+
+- add the new chain case to `getNativeToken()`:
+  - _Example:_ `case 'mythos': return Mainnet.MYTH`
+
+✋ Be careful to schemas when updating any type models. (It must not happen often).<br/>
+=> Check chain swap on the app, sourcechain/destchain filters, sourcechain/tokens filters (UI checks in the transfer form).
+
+**2. 🔐 Env. Variables:** Do not forget to update `NEXT_PUBLIC_PARACHAIN_API_URLS`, in your local env, in the _.env.local.example_ **& Vercel**. ✋ Be aware that adding some _PARACHAIN_API_URLS_ might break the production if not supported (let's remember Muse testnet token/chain with the duplicate parachain Id). Test locally by setting `shouldUseTestnet` constant to **`false`**.
```
Comment (noahjoeris) [inline app/CONTRIBUTING.md:47]:
```suggestion
**2. 🔐 Env. Variables:** Do not forget to update `NEXT_PUBLIC_PARACHAIN_API_URLS`, in your local env, in the _.env.local.example_ **& Vercel**. ✋ Be aware that adding a duplicate parachain Id to _PARACHAIN_API_URLS_ might break production. Test locally by setting `shouldUseTestnet` constant to **`false`**.
```

**Review by noahjoeris — APPROVED** [review]

File: app/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Contributing to Turle
+
+<a id="readme-top"></a>
+
+Turtle is developed with Typescript, Nextjs, and React.
+
+## Contribution Model
+
+The `main` branch refers to the last stable version of <a href="https://app.turtle.cool"><strong>Turtle app</strong></a>. Each pull request should be initiated against the `main` branch and must be approved by at least, one member of the tech team.
```
Comment (NunoAlexandre) [inline app/CONTRIBUTING.md:9]:
```suggestion
The `main` branch refers to the last stable version of <a href="https://app.turtle.cool"><strong>Turtle app</strong></a>. Each pull request should be initiated against the `main` branch and must be approved by at least one member of the tech team.
```

File: app/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+This project uses recommended ESLint and Typescript rules to ensure coding good practices.
+
+We've setup linters and formatters to help catch errors and improve the development experience:
+
+- [Prettier](https://prettier.io/) – ensures that code is formatted in a readable way.
+- [ESLint](https://eslint.org/) — checks code for antipatterns as well as formatting.
+
+**All modifications** must be made in a **pull-request** to solicit feedback from other contributors.
+
+## Information, contributors might be looking for
```
Comment (NunoAlexandre) [inline app/CONTRIBUTING.md:22]:
```suggestion
## Guides
```

File: app/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+### How to support a new parachain
+
+_Parachain prerequisites coming soon._
+
+**1. 📖 Update registry:** (_To support the new chain and tokens on the application_)
+
+- add chain or update existing chain with all the required info from the chain type: find a reliable source for the logo, check the supported address type (evm, or substrate), destinationFeeDOT, specName, rpcConnection, skipExistentialDepositCheck, maxConsumers (…)
```
Comment (NunoAlexandre) [inline app/CONTRIBUTING.md:30]:
```suggestion
- add chain or update existing chain with all the fields required by the chain type: find a reliable source for the logo, check the supported address type (evm, or substrate), destinationFeeDOT, specName, rpcConnection, skipExistentialDepositCheck, maxConsumers, etc.
```

File: app/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  => Some missing information can be found on [Snowbridge SDK](https://github.com/Snowfork/snowbridge/blob/main/web/packages/api/src/environment.ts) or [AT API Registy.](https://github.dev/paritytech/asset-transfer-api-registry/blob/main/docs/registry.json)
+
+- add token if needed: again, find the logo from a reliable source, data (token address or an empty string if not available, multilocation (…))
```
Comment (NunoAlexandre) [inline app/CONTRIBUTING.md:33]:
```suggestion
- add token if needed: again, find the logo from a reliable source, data (token address or an empty string if not available, multilocation, etc)
```

File: app/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+  - _Example:_ `case 'mythos': return Mainnet.MYTH`
+
+✋ Be careful to schemas when updating any type models. (It must not happen often).<br/>
+=> Check chain swap on the app, sourcechain/destchain filters, sourcechain/tokens filters (UI checks in the transfer form).
+
+**2. 🔐 Env. Variables:** Do not forget to update `NEXT_PUBLIC_PARACHAIN_API_URLS`, in your local env, in the _.env.local.example_ **& Vercel**. ✋ Be aware that adding some _PARACHAIN_API_URLS_ might break the production if not supported (let's remember Muse testnet token/chain with the duplicate parachain Id). Test locally by setting `shouldUseTestnet` constant to **`false`**.
+
+**3. ⚙️ Update Polkadot API/PAPI configuration:** (To fetch the balances)
+
+- codegen metadata: add the chain with `npx papi add chainName -w wss://blabla…`
```
Comment (NunoAlexandre) [inline app/CONTRIBUTING.md:51]:
```suggestion
- codegen metadata: add the chain with `npx papi add chainName -w wss://example-endpoint…`
```

**Review by NunoAlexandre — COMMENTED** [review]
Nice :) just a few comments on language and it's good to go!

**Pull request merged**